### DISCUSSION TITLE
Recording chats predictions / placing bets based on chats predictions.

### DISCUSSION THREAD

**Discussion by be-west** [RakambdaOrg/ChannelPointsMiner#258] [Show and tell]
Hi,
first of all, thanks for the project.

I've been working on [my own fork](https://github.com/15sawyer/ChannelPointsMiner/tree/main) that is adding the capability to record other people's predictions to the DB as well as adding a new outcome picker. This outcome picker selects the "most trusted" outcome based on the average Return Of Investment of other people (requires already some data gathered beforehand).

Calculation: After losing a bet, a users ROI is decremented by 1. After winning a bet a users ROI is incremented by (totalPointsOnWinningOutcome / totalPointsOnAllOutcomes).

Predictions are recorded from two sources, from the event updates (these contain only the top predictors) and from chat.

I wanted to ask if there's any desire to add this to the project?

Cheers

**Comment by Rakambda** [comment]
Why not, I'm not against adding it as it's "just" another outcome picker basically.

**Reply by be-west** [reply]
Alright, I'll make a pull request
